src/Render/Markdown/TemplateRenderer.cs

```diff
@@ -0,0 +1,20 @@
+﻿using Scriban;
+using System.Collections.Generic;
+using System.IO;
+using System.Threading.Tasks;
+
+namespace D2L.Dev.Docs.Render.Markdown {
+	internal static class TemplateRenderer {
+	
+		public static async Task<string> RenderAsync( 
+			string templateFileName, 
+			IDictionary<string, string> args 

```

Comment:
Let's not use `IDictionary` here. This function could know about the args we actually have (for now just title and contents) and act as an interface to Scriban which takes a dictionary (i.e. form the dictionary in here rather than Program.cs - Program.cs does not need to know about Scribans API)

Reply:
I suppose we could add an overload to `RenderAsync` that takes a dictionary if we ever need it.